src/main/java/jenkins/plugins/office365connector/WebhookJobPropertyDescriptor.java

```diff
@@ -90,15 +88,19 @@ public WebhookJobProperty newInstance(StaplerRequest req, JSONObject formData) t
         return notificationProperty;
     }
 
-    public FormValidation doCheckUrl(@QueryParameter(value = "url", fixEmpty = true) String url) {
-    	try {
-			new URL(url);
-		} catch (MalformedURLException e) {
-			return FormValidation.error(e.getMessage());
-		}
-		return FormValidation.ok();
+    public FormValidation doCheckUrl(@QueryParameter String value) {
+        if (StringUtils.isBlank(value)) {
+            return FormValidation.error("URL must be provided");
+        }
+
+        try {
+            new URL(value);
```

Comment:
... and then make validation so from now it should store only valid url (null or empty string are not good example of valid url)